doc/overview.rst

```diff
@@ -58,19 +58,19 @@ deploying HPC workloads alongside an in-memory database: Redis.
 
 The key features of the IL are:
 
-  - An API to start, monitor, and stop HPC jobs from Python or Jupyter notebook.
+  - An API to start, monitor, and stop HPC jobs from a Python or Jupyter notebook.
```

Comment:
change to "from Python or from a Jupyter notebook"

Reply:
Will do!